woocommerce/woocommerce
# Fix the visibility icon

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #40174



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Make sure feature `New product editor` is enabled under `/wp-admin/admin.php?page=wc-settings&tab=advanced&section=features`
2. Make sure feature `product-variation-management` is enabled under `Features` tab from `/wp-admin/tools.php?page=woocommerce-admin-test-helper` (WooCommerce Beta Tester plugin) is required
3. Go to `/wp-admin/admin.php?page=wc-admin&path=/add-product&tab=variations` and add some attributes.
4. Then edit the attribute and make sure the `Show in product details` checkbox is unchecked.
5. Then the `eye` icon in the `Variation options` list should be shown like that 
<img width="665" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/942a686b-431c-4d3f-ad4d-0b25b309a348">

6. As well as in the `Variations` list 
<img width="663" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/84d5f718-349c-402e-b1bb-d9dbd3e9037a">

7. Go to `/wp-admin/admin.php?page=wc-admin&path=/add-product&tab=organization` and add some attributes, make sure the `Show in product details` checkbox is unchecked.
8. Then the `eye` icon in the `Attributes` list should be shown like that 
<img width="667" alt="image" src="https://github.com/woocommerce/woocommerce/assets/13334210/57f2c2e7-e011-45b2-8622-c916e3ae6374">




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
